modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/extension/v1_0/ObjectEntryExtensionProvider.java

```diff
@@ -61,13 +70,32 @@ public Map<String, Serializable> getExtendedProperties(
 					getExtensionDynamicObjectDefinitionTableValues(
 						objectDefinition, getPrimaryKey(entity));
 
+			JSONObject jsonObject = jsonFactory.createJSONObject(
+				jsonFactory.looseSerializeDeep(entity));
+
+			Map<String, Serializable> variables =
```

Comment:
I noticed that you use this var only inside 1 IF. Could you declare this variable inside the condition that you use?

Reply:
resolved.